### ISSUE TITLE
[Feature]: Add SimpleX to Accrescent

### ISSUE THREAD

**Issue by An-anonymous-coder** [simplex-chat/simplex-chat#5012]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Android

### App version

N/A

### Feature

SimpleX should be available to install from [Accrescent](https://accrescent.app/), due to it being one of the most secure methods for installing apps according to the GrapheneOS team. 

[This issue](https://github.com/simplex-chat/simplex-chat/issues/3994) brought this up but was self-closed due to the author claiming "Accrescent is not just open source app store, but it host also closed source apps". More context is given in [this issue](https://github.com/accrescent/accrescent/issues/479) opened by the same user. The user closed their original issue simply because they do not like Accrescent, so I am reopening it with more details.

For what it's worth, all apps released to Accrescent so far are open source. Here are all the apps currently available for Accrescent with links to their source code as evidence:

[Accrescent](https://github.com/accrescent/accrescent)

[Organic Maps: Offline Map](https://github.com/organicmaps/organicmaps)

[Just (Video) Player](https://github.com/moneytoo/Player)

[Arcticons](https://github.com/Arcticons-Team/Arcticons)

[Arcticons Black](https://github.com/Arcticons-Team/Arcticons)

[Arcticons Material You](https://github.com/Arcticons-Team/Arcticons)

[Clipious](https://github.com/lamarios/clipious)

[ExifEraser](https://github.com/Tommy-Geenexus/exif-eraser)

[trale](https://github.com/QuantumPhysique/trale)

[Aves Gallery](https://github.com/deckerst/aves)

[Aves Gallery Libre](https://github.com/deckerst/aves)

[AppVerifier](https://github.com/soupslurpr/AppVerifier)

[BeauTyXT](https://github.com/soupslurpr/BeauTyXT)

[Transcribro](https://github.com/soupslurpr/Transcribro)

[Molly](https://github.com/mollyim/mollyim-android)

[IVPN - Secure VPN for Privacy](https://github.com/ivpn/android-app)

[RiMusic](https://github.com/fast4x/RiMusic)

[Blitz: Fischer Chess Clock](https://github.com/ldeso/blitz)

*This is not meant to advertise any of these projects, I am just including them here because of why the user closed the original issue.

**Comment by PrivacyAndSecurity** [comment]
I agree with him, that we would love to see SimpleX Chat on Accrescent! Maybe you ask why we want this or what Benefits it would have for us as Users. 

The Benefits for me would be a few, like these:

- I could get SimpleX Chat from Accrescent, which I already installed and use to download Apps. I prefer Accrescent over Github, because on Accrescent the App get's checked before it comes to the Store and the Apps are always checked, like for example the SDK of Apps, if there is Malware and more.

- It would make it a lot easier for me, because SimpleX Chat is one of the few Apps, I use in Obtainium to download. And if my last few Apps would come to Accrescent, I wouldn't need to use Obtainium. I just use it until my last few Apps on there come to Accrescent.

- Apps on Github are not checked, but on Accrescent they are going trough Filters always. And I think that would make it more secure. And since Accrescent is on a good way in terms of Security and Privacy, I think that it will be really good in the Future. And I think that Apps which value Security and Privacy should make it available there. Because Accrescent have the same values in Security and Privacy.


I really hope that you Guys could make that possible, because it would have great Benefits for us as Users.

**Comment by randomwithnoname** [comment]
> I could get SimpleX Chat from Accrescent, which I already installed and use to download Apps. I prefer Accrescent over Github, because on Accrescent the App get's checked before it comes to the Store and the Apps are always checked, like for example the SDK of Apps, if there is Malware and more.

I don't think Accrescent runs antivirus. Apps are manually reviewed when they're first published as well as when they request new sensitive permissions.

> It would make it a lot easier for me, because SimpleX Chat is one of the few Apps, I use in Obtainium to download. And if my last few Apps would come to Accrescent, I wouldn't need to use Obtainium. I just use it until my last few Apps on there come to Accrescent.

It would certainly be a lot easier.

> Apps on Github are not checked, but on Accrescent they are going trough Filters always. And I think that would make it more secure. And since Accrescent is on a good way in terms of Security and Privacy, I think that it will be really good in the Future. And I think that Apps which value Security and Privacy should make it available there. Because Accrescent have the same values in Security and Privacy.

You're still always trusting the app developer, app store review adds little to no security. The real benefits are listed here: https://accrescent.app/#features

Accrescent's security is already extended to SimpleX thanks to the AppVerifier app (https://github.com/soupslurpr/AppVerifier)You can check if your SimpleX install from elsewhere is genuine.

Also worth linking this: https://github.com/accrescent/parcelo/issues/635

**Comment by PrivacyAndSecurity** [comment]
Is there any Updates on this?

**Comment by randomwithnoname** [comment]
https://github.com/accrescent/parcelo/issues/635

> Is there any Updates on this?

**Comment by PrivacyAndSecurity** [comment]
> [accrescent/parcelo#635](https://github.com/accrescent/parcelo/issues/635)
> 
> > Is there any Updates on this?

But there is no really Update, if SimpleX will come to Accrescent. It just stopped there.

**Comment by randomwithnoname** [comment]
> > [accrescent/parcelo#635](https://github.com/accrescent/parcelo/issues/635)
> > > Is there any Updates on this?
> 
> But there is no really Update, if SimpleX will come to Accrescent. It just stopped there.

Accrescent plans to add generic username/password option. Once that's done, there isn't much reason not for it not to.

**Comment by PrivacyAndSecurity** [comment]
> > > [accrescent/parcelo#635](https://github.com/accrescent/parcelo/issues/635)
> > > > Is there any Updates on this?
> > 
> > 
> > But there is no really Update, if SimpleX will come to Accrescent. It just stopped there.
> 
> Accrescent plans to add generic username/password option. Once that's done, there isn't much reason not for it not to.

Oh, what does that mean with generic username/password? Okay nice then.

**Comment by randomwithnoname** [comment]
> > > > [accrescent/parcelo#635](https://github.com/accrescent/parcelo/issues/635)
> > > > > Is there any Updates on this?
> > > 
> > > 
> > > But there is no really Update, if SimpleX will come to Accrescent. It just stopped there.
> > 
> > 
> > Accrescent plans to add generic username/password option. Once that's done, there isn't much reason not for it not to.
> 
> Oh, what does that mean with generic username/password? Okay nice then.

You will be able to create an Accrescent developer account without needing to share your private GitHub emails.

**Comment by randomwithnoname** [comment]
> Accrescent accepts both closed source and open source app.
> 
> Definitely it is not my style or need. 👎
> 
> I don't confuse pumpkins for carrots just because both are orange.

This really isn't the place to discuss this. 

I don't like proprietary software as much as the next person but Accrescent is doing a public good overall and will ultimately benefit open source Android apps by providing an easy and secure way to obtain them as well as much needed competition to Google Play. 

They will add filter options including filtering by open source apps.
